src/org/zaproxy/zap/extension/technology/TechnologyAPI.java

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
The new API should be added to ApiGeneratorUtils so it gets generated for the (Java, Python, ...) API clients.
Good to have some docs (added to Messages.properties file, the keys can be checked by running, for example, the Java API generator).